### PR TITLE
python: add "bpdb" snippets for bpython

### PR DESCRIPTION
**Pull request by blueyed**

### CODE REVIEW

File: UltiSnips/python.snippets [hunk 1]
```diff
@@ -634,6 +634,11 @@ snippet ipy "Embed IPython shell" b
 import IPython; IPython.embed()
 endsnippet
 
+snippet bpdb "Set bpython/bpdb breakpoint" b
+import bpdb
+bpdb.set_trace()
+endsnippet
```
Comment (lpil) [inline UltiSnips/python.snippets]: No need to add it here if it is in the snipmate format already :)

Comment (blueyed) [inline UltiSnips/python.snippets]: That applies to the others then also, right?
Is `snippets` / snipmate the preferred one?  I would have assumed that `UltiSnips` would be preferred?!

Comment (lpil) [inline UltiSnips/python.snippets]: Snippets is the preferred as Ultisnips can read both formats :)

Comment (blueyed) [inline UltiSnips/python.snippets]: But then snippets appears to have less features..?!

Comment (lpil) [inline UltiSnips/python.snippets]: You're not using any of those features.

**Comment by blueyed** [comment]
Closing in favour of #676.

**Pull request closed without merging**